ports/_resources/port1.0/fetch/archive_sites.tcl

```diff
@@ -0,0 +1,23 @@
+namespace eval portfetch::mirror_sites { }
+global os.platform os.major archivefetch_pubkeys
+set portfetch::mirror_sites::sites(macathena_archives) https://macathena.mit.edu/dist/port_archives/:nosubdir
+set portfetch::mirror_sites::archive_type(macathena_archives) tbz2
```

Comment:
Maybe leave a comment here pointing to the file that we copied all of these values from, and saying that they need to match or the archive won't be used?